slapp/transforms/image_utils.py

```diff
@@ -0,0 +1,71 @@
+from typing import Tuple, Union
+from pathlib import Path
+from math import ceil
+
+import numpy as np
+from PIL import Image, ImageDraw, ImageFont
+
+
+def add_scale(image: Union[Path],
+              scale_position: Tuple[int, int],
+              font_file: Path,
+              resolution: float = 10 / 13,
+              scale_size: int = 10,
+              fill_color: Tuple[int, int, int] = (0, 0, 0),
+              line_weight: int = 5,
+              font_size: int = 8) -> Image:
+    """
+    Adds a scale bar onto a image passed in the form of a image path. Uses
+    PIL library ImageDraw and ImageFont to draw scale bar onto images.
+    Parameters
+    ----------
+    image: Path
+        path to the image to add scale bar
+    scale_position: Tuple
+        the position at which to set the bottom left corner of the scale. In
+        x, y format, this is the format PIL expects.
+    font_file: Path
+        the font file to be used as denoted by a path object. On windows can
+        just be specified as 'arial.ttf' on linux must be a full path.
+    resolution: float = 10/13
+        resolution of the video in microns per pixel, defaulted to
+        scientifica general which is 10/13~.78 micros/pixel
+    scale_size: int=10
+        how many micro meters the scale bar should display, defaulted to 10
+    fill_color: Tuple=(0, 0, 0)
```

Comment:
for `fill_color` and `line_weight` you are passing these right into `draw()`, I think it makes sense not to rename them to be exactly the same as the keywords for `draw()` and to note in your docstring that they are being directed to those keywords in draw(). Then you don't have to redefine them. Reader can refer to docs for `draw()`.

Reply:
I see this is in direct opposition to what @kschelonka asked for earlier. I disagree, but you don't have to act on this one.

Reply:
@djkapner In general I agree with trying to keep the parameters. If it were explicitly mentioned in the docstring I think it would be fine. The main reason I asked for changes is because I think those names make sense for the line drawing function, but are more confusing in the sense of a scale drawing -- especially "width".

Reply:
I'll revert these and have the params reflect the PIL params.